inference-engine/ie_bridges/java/sample/face_detection_java_sample/README.md

```diff
@@ -0,0 +1,54 @@
+# Face Detection Java Sample
+
+This guide describes how to run the java Face Detection sample application.
+
+## How It Works
+
+Upon the start-up the sample application reads command line parameters and loads a network and an image to the Inference
+Engine device. When inference is done, the application creates an output image and outputs data.
+
+To download model ( .bin and .xml files must be downloaded) use:
+https://download.01.org/opencv/2019/open_model_zoo/R1/models_bin/face-detection-adas-0001/FP32/
+
+## Build
+To create java library for Inference Engine add `-DENABLE_JAVA=ON` flag in cmake command while building dldt:
+```bash
+cmake -DCMAKE_BUILD_TYPE=Release -DENABLE_JAVA=ON ..
+```
+
+Create an environment variable with OpenCV installation path:
+```bash
+export OCV_PATH=/path/to/opencv/build
+```
+
+and the same for Inference Engine:
+```bash
+export IE_PATH=/path/to/openvino/bin/intel64/Release/lib
+```
+
+Then compile parser class:
+```bash
+cd /path/to/openvino/inference-engine/ie_bridges/java/sample/
+javac -d face_detection_java_sample ArgumentParser.java
+```
+
+To compile sample use:
+```bash
+cd face_detection_java_sample
+javac -cp ".:${OCV_PATH}/bin/opencv-420.jar:${IE_PATH}/inference_engine_java_api.jar" Main.java
+```
+
+## Running
+Add library path for openvino java library and for opencv library before running:
+```bash
+export LD_LIBRARY_PATH=${OCV_PATH}/lib:/${IE_PATH}:$LD_LIBRARY_PATH
+```
+
+To run sample use:
+```bash
+java -cp ".:${OCV_PATH}/bin/opencv-420.jar:${IE_PATH}/inference_engine_java_api.jar" Main -i <image_path> -m face-detection-adas-0001.xml
+```
+
+## Sample Output
+
+The application will show the image with detected objects enclosed in rectangles in new window. It outputs the confidence value and the coordinates of the rectangle to the standard output stream.
```

Comment:
Let's avoid duplication of README files and make a single one at ` inference-engine/ie_bridges/java/sample/README.md`